ivy_tests/test_ivy/test_misc/test_exceptions.py

```diff
@@ -3,7 +3,7 @@
 import contextlib
 import pytest
 from ivy_tests.test_ivy.helpers.available_frameworks import available_frameworks
-
+import ivy.functional.frontends.torch as torch
```

Comment:
I don't think we should import the frontends for these tests, frontend-specific exception tests would be present in the frontend files, as the frontends would have their own exception classes, as directed by the corresponding framework. We should instead be using `ivy` functions inside the `test_trace_modes`